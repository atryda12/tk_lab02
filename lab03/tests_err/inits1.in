int a = 3, b=12 c="lalala"