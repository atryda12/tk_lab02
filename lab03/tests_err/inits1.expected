"Invalid definition of variable 'c': Expected: 'int', got: 'string'